### DISCUSSION TITLE
Other Materials Check

### DISCUSSION THREAD

**Discussion by isBlake** [Zaki-Mao/MUST-English-Debate-Team-EDT-#7] [Q&A] [answered]
for BP debate

**Comment by Zaki-Mao** [comment, marked as answer]
BP Debate Intro Video https://www.bilibili.com/video/BV1Jt411C7K8
Oxford Union Debate https://oxford-union.org/pages/debating
Debate Motions https://debate-motions.info/
Debate Motions -2 http://hellomotions.com/
